backend/src/models/nativeLanguage.js

```diff
@@ -0,0 +1,44 @@
+const { Model } = require('objection')
+const tableNativeLanguages = 'nativeLanguages'
+
+class NativeLanguage extends Model {
+  static get tableName () {
+    return tableNativeLanguages
+  }
+
+  static get idColumn () {
+    return ['NativeLanguageID']
+  }
+
+  static get relationMappings () {
+    const Student = require('./student')
+
+    return {
+      nativeLanguage: {
```

Comment:
```suggestion
      students: {
```
We are retrieving a list of `Student`s with a given `NativeLanguageID`.